common/src/main/java/com/movie/recommender/common/model/MovieShowtimesResponse.java

```diff
@@ -0,0 +1,66 @@
+package com.movie.recommender.common.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MovieShowtimesResponse {
+
```

Comment:
since `com.movie.recommender.common.model` now contains many classes, create two packages inside it: `movie` and `showtime`, and move all classes accordingly